tsconfig.json

```diff
@@ -4,13 +4,14 @@
     "module": "commonjs",
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "target": "es6",
+    "target": "es5",
```

Comment:
whats the reasoning for this? es6 would be best would it not?

Reply:
It would :( However unfortunately IE11 only supports ES5, therefore needed to lower it

Reply:
hmm that brings us 6 years out of date, but fair enough if thats the only option

Reply:
There are pollyfills which transpile es6 code into vanilla javascript, rather than downgrading maybe we should try to use them?
Might be the matter of import only? With Angular and React there is usually pollyfills file and you just need to uncomment lines to enable it
https://polyfill.io/v3/url-builder/

Reply:
https://github.com/commitBlob/folioAngular/blob/master/src/polyfills.ts
This is Angular example

Reply:
i saw something similar, this is another option https://babeljs.io/

Reply:
Ah nice! I'll have a look at those now